app/org/sagebionetworks/bridge/stormpath/StormpathDirectoryDao.java

```diff
@@ -158,4 +198,109 @@ private Directory getDirectory(String name) {
         DirectoryList list = client.getDirectories(criteria);
         return (list.iterator().hasNext()) ? list.iterator().next() : null;
     }
+    
+    private void adjustPasswordPolicies(Study study, Directory directory) {
+        EmailTemplate rp = study.getResetPasswordTemplate();
+        
+        PasswordPolicy passwordPolicy = directory.getPasswordPolicy();
+        passwordPolicy.setResetEmailStatus(EmailStatus.ENABLED);
+        passwordPolicy.setResetSuccessEmailStatus(EmailStatus.DISABLED);
+        
+        ModeledEmailTemplateList resetEmailTemplates = passwordPolicy.getResetEmailTemplates();
+        
+        // According to the documentation, there is only one...
+        ModeledEmailTemplate template = resetEmailTemplates.iterator().next();
+        template.setFromName(study.getSponsorName());
+        template.setFromEmailAddress(study.getSupportEmail());
+
+        String subject = partiallyResolveTemplate(rp.getSubject(), study);
+        template.setSubject(subject);
+        
+        com.stormpath.sdk.mail.MimeType stormpathMimeType = getStormpathMimeType(study);
+        template.setMimeType(stormpathMimeType);
+        
+        String body = partiallyResolveTemplate(rp.getBody(), study);
+        template.setTextBody(body);
+        template.setHtmlBody(body);
+
+        String link = String.format("%s/mobile/resetPassword.html?study=%s", config.getBaseURL(), study.getIdentifier());
+        template.setLinkBaseUrl(link);
+        template.save();
+        
+        PasswordStrength strength = passwordPolicy.getStrength();
+        strength.setMaxLength(100);
```

Comment:
Added this (also it's now 999, Stormpath was setting a default of 100, can be higher, but needs to be some value).